### PR TITLE
added the documentation part

### PR DESCRIPTION
**Pull request by lannisterrr**
Added the documentation part for my component library.

### CODE REVIEW

File: documentation/navbar.css [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+}
+
+.title {
+  padding: 0rem 1rem;
+  color: var(--primary-color);
+  margin-left: -5rem;
+}
+
+@media only screen and (max-width: 600px) {
+  /* max width */
```
Comment (Mohasin-Haque) [inline documentation/navbar.css:84]: Use all Media queries at one place so it will be easier to maintain it

**Review by Mohasin-Haque — COMMENTED** [review]
Everything seems fine to me but please take care as mentioned in the comments

**Review by NtshVrm — APPROVED** [review]
Looks good👏

File: documentation/navbar.css (hunk 1 above)
Comment (lannisterrr) [reply documentation/navbar.css:84]: ok thank you.

**Pull request merged**